SI/Sem.02/Pract.02/Task6.cpp

```diff
@@ -0,0 +1,34 @@
+
+
+#include <iostream>
+using namespace std;
+int main()
+{
+	double a, b, c, x1, x2, D;
+	cout << "a = ";
+	cin >> a;
+	cout << "b = ";
+	cin >> b;
+	cout << "c = ";
+	cin >> c;
+	D = sqrt(b * b - 4 * (a * c));
```

Comment:
D = b * b - 4 * (a * c);
You have to check whether D < 0 before you use sqrt, because you can only calculate square root of non negative numbers. Then, after the checking, you can calculate sqrt(D).